modules/ztp-sno-du-enabling-workload-partitioning.adoc

```diff
@@ -57,16 +57,16 @@ sh-4.4# pgrep ovn | while read i; do taskset -cp $i; done
 .Example output
 [source,terminal]
 ----
-pid 8481's current affinity list: 0-3
-pid 8726's current affinity list: 0-3
-pid 9088's current affinity list: 0-3
-pid 9945's current affinity list: 0-3
-pid 10387's current affinity list: 0-3
-pid 12123's current affinity list: 0-3
-pid 13313's current affinity list: 0-3
+pid 8481's current affinity list: 2-51,54-102
+pid 8726's current affinity list: 2-51,54-102
+pid 9088's current affinity list: 2-51,54-102
+pid 9945's current affinity list: 2-51,54-102
+pid 10387's current affinity list: 2-51,54-102
+pid 12123's current affinity list: 2-51,54-102
+pid 13313's current affinity list: 2-51,54-102
```

Comment:
They should all says `0-1,52-53` since `ovn` is also pinned to the small set of CPUs (as they are OCP apps) just like the system apps (e.g `systemd`)